polaris-react/src/components/IndexTable/IndexTable.scss

```diff
@@ -216,7 +216,19 @@ $loading-panel-height: 53px;
 
       #{$se23} & {
         // Bump specificity
-        background-color: var(--p-color-bg-interactive-selected);
+        background-color: var(--p-color-bg-subdued-active);
```

Comment:
**suggestion:** Can remove the above comment since this is no longer purely a specificity bump.